main.go

```diff
@@ -59,6 +61,7 @@ func main() {
 	auditOutputURL := flag.String("output-url", "", "Destination URL to send audit results")
 	auditOutputFile := flag.String("output-file", "", "Destination file for audit results")
 	auditOutputFormat := flag.String("output-format", "json", "Output format for results - json, yaml, or score")
+	loadAuditFile := flag.String("load-audit-file", "", "Runs the dashboard with data saved from a past audit.")
```

Comment:
Maybe not a good comment for this PR: but noteworthy. The arguments for this one binary are getting very complex and intertwined. It's technically possible to set `-load-audit-file` with the `-webhook` flag and it will just ignore the first flag. This (to me) is confusing and maybe could leverage some subcommand support in the future. Again not for this PR but something to consider a future change for.

Reply:
100% agree. https://github.com/FairwindsOps/polaris/issues/190